source/class/Game.es6

```diff
@@ -7,12 +7,22 @@
 	let Clock = require('./Clock.es6');
 	let Persist = require('./Persist.es6');
 	let RNG = require('./RNG.es6');
+	let Level = require('./Level.es6');
 
 	module.exports = class Game extends Persist {
-		constructor() {
-		},
+		constructor(id, dflt) {
```

Comment:
source code self doc, what is dflt? rename to something more meaningful?

Reply:
Yeah, the persist constructor needs some cleaning up.  I will work on that.